core/src/io/anuke/mindustry/world/blocks/power/PowerNode.java

```diff
@@ -281,9 +281,13 @@ public void drawLayer(Tile tile){
 
         for(int i = 0; i < entity.power.links.size; i++){
             Tile link = world.tile(entity.power.links.get(i));
-            if(linkValid(tile, link) && (link.pos() < tile.pos() || !(link.block() instanceof PowerNode) || !Core.camera.bounds(Tmp.r1).contains(link.drawx(), link.drawy()))){
-                drawLaser(tile, link);
-            }
+            if (!linkValid(tile, link))
```

Comment:
Unnecessary space after `if`, `continue` should be on the same line as the if-statement

Reply:
Done. Though I think it makes code less readable.